.kitchen.yml

```diff
@@ -15,37 +15,51 @@ verifier:
   name: inspec
 
 platforms:
-- name: fedora-latest
-  driver:
-    image: fedora:latest
-    pid_one_command: /usr/lib/systemd/systemd
-    intermediate_instructions:
-      - RUN dnf -y install yum which net-tools procps-ng telnet iputils nmap
-  run_list:
-    - recipe[chef-yum-docker]
+  - name: hello
+    driver:
+      image: alpine:latest
```

Comment:
Example "arbitrary" container -- this is representative of an  in-house "fooapp" container.